### PR TITLE
ci: add new release workflow

### PR DESCRIPTION
**Pull request by splincode**
@nsbarsukov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,11 +55,7 @@
         "cy:open": "cypress open --project ./projects/demo-integrations/",
         "cy:run": "nx e2e demo-integrations",
         "*** Release ***": "",
-        "release": "standard-version -a --no-verify",
-        "release:patch": "npm run release -- --release-as patch",
-        "release:minor": "npm run release -- --release-as minor",
-        "release:major": "npm run release -- --release-as major",
-        "publish": "nx run-many --target build --all --exclude=demo && nx run-many --target publish --all"
+        "bump": "npx syncer && npx nx run-many --target publish --all"
```
Comment (nsbarsukov) [inline package.json:58]:
```suggestion
        "publish": "npx syncer && npx nx run-many --target publish --all"
```

File: package.json [hunk 2]
```diff
@@ -55,11 +55,7 @@
         "cy:open": "cypress open --project ./projects/demo-integrations/",
         "cy:run": "nx e2e demo-integrations",
         "*** Release ***": "",
```
Comment (nsbarsukov) [inline package.json:57]: Remove this line

Comment (splincode) [reply package.json]: fixed

File: package.json (hunk 1 above)
Comment (splincode) [reply package.json]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**